elements/rh-icon/README.md

```diff
@@ -1,5 +1,53 @@
 # WIP 🐣: RHElements Icon Element
 
+## Usage
+
+```
+<rh-icon icon="rh-icon-server"></rh-icon>
+```
+
+## Attributes
+
+### icon (observed)
+
+The name of the icon that you want to use. If the value of this attribute changes, the new icon will show up in the UI.
+
+### data-size
+
+Controls the size of the icon. The options for size are:
+- `2x`
+- `3x`
+- `4x`
+- `small`
+- `medium`
+- `large`
+
+```
+<rh-icon icon="rh-icon-server" data-size="2x"></rh-icon>
+```
+
+### data-block
+
+By default, `rh-icon` is set to `display: inline-block`. The `data-block` attribute sets this element to `display: block`.
+
+```
+<rh-icon icon="rh-icon-server" data-block></rh-icon>
+```
+
+### data-bg
```

Comment:
I'm going off the rails of this PR a bit, but I find it a little confusing that an attribute named `bg` can control both the color of the background (transparent or not) and also the shape (circle or square).